lib/mayaUsd/sceneIndex/proxyShapeSceneIndexPlugin.cpp

```diff
@@ -397,7 +397,7 @@ HdSceneIndexPrim MayaUsdProxyShapeSceneIndex::GetPrim(const SdfPath& primPath) c
 
 SdfPathVector MayaUsdProxyShapeSceneIndex::GetChildPrimPaths(const SdfPath& primPath) const
 {
-    return _usdImagingStageSceneIndex->GetChildPrimPaths(primPath);
+    return _GetInputSceneIndex()->GetChildPrimPaths(primPath);
```

Comment:
Great catch!  Can we provide an automated test that will prevent regressions?

Reply:
Yep, working on it, though it'll be on the MayaHydra side. I've been stripping down the ALab scene to get a minimal repro scene.

Reply:
This is why we need to pull all the rendering-related USD code into maya-hydra...